src/goInstallTools.ts

```diff
@@ -112,7 +112,7 @@ export function installTools(missing: Tool[], goVersion: SemVersion): Promise<vo
 	// This ensures that users get the latest tagged version, rather than master,
 	// which may be unstable.
 	let modulesOff = false;
-	if (isBelow(goVersion, 1, 11)) {
+	if (semver.lt(goVersion, '1.11.0')) {
```

Comment:
How does `semver.lt` behave when the first argument is null?
Same question for `gt` or other comparison functions that are being used.

In our case `goVersion` will be null when using Go from the tip
You will need to add a null check on `goVersion` every time we use the `lt`, `gt` function

Reply:
I reworked some of the Go version handling to tackle developer versions. There's definitely more that can be done to improve handling, but this should work for now.